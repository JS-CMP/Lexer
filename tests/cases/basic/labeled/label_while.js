loop: while (true) {
    console.log("loop");
    break loop;
}

