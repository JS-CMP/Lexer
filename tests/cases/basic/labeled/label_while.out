#include "types/JsAny.hpp"
#include "global/global.hpp"
#include "types/objects/Types.hpp"
#include "customOperators/CustomOperators.hpp"

int main() {
loop:
    while (JS::Any(true)) 
    {
        console[u"log"](JS::Any(u"loop"));
        break /* label: ; JS_CMP does not support labeled break for now */;
    }
    return 0;
}
